spec/features/merchants/items/index_spec.rb

```diff
@@ -39,6 +39,47 @@
         click_link "#{item_2.name}"
         expect(current_path).to eq("/merchants/#{merchant_1.id}/items/#{item_2.id}")
       end
+
+      # 9. Merchant Item Disable/Enable
+
+      # As a merchant
+      # When I visit my items index page (/merchants/:merchant_id/items)
+      # Next to each item name I see a button to disable or enable that item.
+      # When I click this button
+      # Then I am redirected back to the items index
+      # And I see that the items status has changed
+
+      it "displays a button that will enable or disable each item" do
+        visit "/merchants/#{merchant_1.id}/items"
+        within ".enabled-items" do
+          expect(page).to have_content("Enabled Items")
+          expect(page).to have_link(item_1.name)
+          expect(page).to have_link(item_2.name)
+          expect(page).to have_link(item_3.name)
+
+          expect(page).to have_button("Disable #{item_1.name}")
+          expect(page).to have_button("Disable #{item_2.name}")
+          expect(page).to have_button("Disable #{item_3.name}")
+
+          click_button "Disable #{item_1.name}"
+        end
+
+        expect(current_path).to eq("/merchants/#{merchant_1.id}/items")
+
+        within ".disabled-items" do
+          expect(page).to have_content("Disabled Items")
+          expect(page).to have_link(item_1.name)
+          expect(page).to_not have_content(item_2.name)
+          expect(page).to_not have_content(item_3.name)
+
+          expect(page).to have_button("Enable #{item_1.name}")
```

Comment:
Hey Jesse, remember we talked about adding 2 lines under this such as:
`expect(page).to_not have_button("Enable #{item_2.name}")`
`expect(page).to_not have_button("Enable #{item_3.name}")`